src/credentials/CHIPCert.cpp

```diff
@@ -1095,12 +1095,19 @@ DLL_EXPORT CHIP_ERROR ChipEpochToASN1Time(uint32_t epochTime, chip::ASN1::ASN1Un
     return CHIP_NO_ERROR;
 }
 
+static CHIP_ERROR ValidateCertificateType(ChipCertificateData const & certData, CertType expectedType)
```

Comment:
```suggestion
static CHIP_ERROR ValidateCertificateType(const ChipCertificateData & certData, CertType expectedType)
```
is the normal style in Matter, seems like.

Reply:
Die hard, old habits do!